messaging/src/main/java/org/axonframework/eventhandling/TrackingEventProcessor.java

```diff
@@ -402,6 +411,17 @@ private void processBatch(Segment segment, BlockingStream<TrackedEventMessage<?>
         }
     }
 
+    private void logReceivedEvent(Segment segment, TrackedEventMessage<?> message, BlockingStream<?> eventStream) {
+        logger.trace("Processor {} receiving event {}. Tracking token {}. Segment id {}. Thread {}. Processor instance {}. Event Stream {}.",
```

Comment:
Add a `logger.isTraceEnabled` to prevent unnecessary invocation of getter methods (including accessing the current Thread's name)